app/editor/src/features/admin/evening-overview/interfaces/IEveningOverviewItemType.ts

```diff
@@ -0,0 +1,11 @@
+export interface IEveningOverviewItemType {
+  [key: string]: {
+    id: number;
+    name: string;
+  };
+}
+export const EveningOverviewItemType: IEveningOverviewItemType = {
```

Comment:
Why are you not following the same pattern we've used for all of our other implementations?  We use an enum.  The API saves the value as an integer, but for the frontend we use a string value.